### ISSUE TITLE
Everytime I Restart Blender I need to Reload ZBrush Preset

### ISSUE THREAD

**Issue by ManuelGPz** [dairin0d/mouselook-navigation#4]
Blender Version 4.0.2 Portable
Bug show in 
Mouse Navigator 1.9.2 and 1.7.8

As per the title, every time I restart Blender I need to reload the Zbrush Key Preset ( which is the one I use ), otherwise the Alt+LMB drag zoom doesn't work.

I have saved Preferences and Default Startup File with Mouse Navigation Addon Active and Zbruhs Preset and the error persists.

I have also noticed that once the addon is active and the Zbrush Preset is loaded, every time I use Mouse Navigation my System Preferences change somehow, as I can see that the Asterix (*) that shows you there has been a change pops up. 
Doesnt matter how many times I save I need to load the Zbrush preset every time I load Blender.

This happens everytime I open Blender, however if I load the preset, and I open a new file from the current instance of Blender, whatever file I open, a new one, or another one, would work with the Zbrush preset on.
Closing Blender and opening it again would make the Alt+LMB Drag Zoom not Funcional.

Please let me know if there is anything I can provide to help. 

Thanks so much

**Comment by dairin0d** [comment]
Hi! Sorry for the delayed response. Unfortunately, I'm not able to reproduce on my computer the "need to reload ZBrush" issue you're experiencing. Are there any errors in the system console?
Just in case, here's how my keymap looks like with the ZBrush preset. Does it look the same for you when you just opened Blender?
![image](https://github.com/dairin0d/mouselook-navigation/assets/1641842/2fea8713-5076-48fd-94b4-72b92d57562c)

The preferences getting "changed" is unavoidable, unfortunately, because some specific values of certain preferences are needed to operate correctly or to support some features of the addon. The Mouselook Navigation operator reverts those preferences to their original values after the navigation has ended, but Blender would still consider them "changed" and show the asterisk.

**Comment by ManuelGPz** [comment]
Thanks so much for that, and sorry for the late answer. I have been on a trip and I couldnt check this before.

Thanks to your screenshot I found the problem. For I'm not sure what reason, Blender was duplicating the mouse look navigation hotkeys on startup. I guess that was creating some conflict and they were not working.
This is the way the shortcuts look on Startup:
![MouseLook-001](https://github.com/dairin0d/mouselook-navigation/assets/58131591/bc8b00ce-15d7-48d4-a2cc-0168604e815a)

To solve it, I deleted the duplicates, saved the preferences and this time on startup Blender is not creating duplicates again. 

Working like a charm again.

Thanks so much!
